cmd/dlt/admin/cmd.go

```diff
@@ -45,85 +45,93 @@ func run(cmd *cobra.Command, _ []string) {
 	r := rosa.NewRuntime().WithAWS().WithOCM()
 	defer r.Cleanup()
 
-	clusterKey := r.GetClusterKey()
-
 	cluster := r.FetchCluster()
 	if cluster.State() != cmv1.ClusterStateReady {
-		r.Reporter.Errorf("Cluster '%s' is not yet ready", clusterKey)
+		r.Reporter.Errorf("Cluster '%s' is not yet ready", r.ClusterKey)
 		os.Exit(1)
 	}
 
 	// Try to find the htpasswd identity provider:
 	r.Reporter.Debugf("Loading HTPasswd identity provider")
-	idps, err := r.OCMClient.GetIdentityProviders(cluster.ID())
+	clusterID := cluster.ID()
```

Comment:
NP: just wondering if it's not an optimization the compiler makes anyway

Reply:
hmm, probably. But it was more to concentrate it in a single place in case we ever change something